flaml/data.py

```diff
@@ -193,6 +195,9 @@ def fit_transform(self, X, y, task):
             cat_columns, num_columns = [], []
             drop = False
             for column in X.columns:
+                # sklearn\utils\validation.py needs int/float values
+                X.loc[:, column] = X[column].map(datetime.toordinal) if X[column].dtype.name in (
```

Comment:
I suggest a small change:
```python
if X[column].dtype.name == 'datetime64[ns]':
    X[column] = X[column].map(datetime.toordinal)
```
I assumed you meant a tuple when using `('datetime64[ns]')`.

Reply:
Sure, it is better!